spec/models/mobile_user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe MobileUser, type: :model do
+
+  context 'when last_name is short' do
```

Comment:
Dodałbym jeszcze jeden kontekst, żeby było wiadomo co tak naprawdę testujemy.

`context 'identity params' do`
..`context 'when last name consists of only one part' do`
..`context 'when last name consists of two parts' do`